parsers/verilog.c

```diff
@@ -299,6 +307,12 @@ static const char *systemVerilogKeywords[] = {
 	"wildcard", "wire", "with", "within", "wor", "xnor", "xor"
 };
 
+static fieldDefinition VerilogFields[] = {
```

Comment:
In your change, this array is shared between the SystemVerilog parser and the Verilog parser.
However, parsers should not share parser specific fields.
So, define SystemVerilogFields by copying & pasting VerilogFields.